io/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 add_subdirectory(io)  # special CMakeLists.txt
+add_subdirectory(minio)
```

Comment:
Is it intentional to have a difference of one character between the directory name and the library name (missing ```i```)?  Maybe we ought to find a difference prefix to avoid the double/repeated `i`?

Reply:
ops, I will just rename it as `miniio`. I lost one 'i' somewhere...

Reply:
For the name, we have so far `libRIO` and `libRMiniIO`.  In some mind 'mini' could be seems has derogative (maybe). And the lower `i` follower by the uppercase `I` is a bit hard to read.

So a few potential alternatives `libRIOLight`,  `libRIOlight`,`libRCoreIO`, `libRBaseIO`, `libRIOfoundation` ..

@jblomer @Axel-Naumann Any better ideas and/or favorite choice?

Reply:
ROOT I/O meeting concluded with: `libROOTIOLite`